R/tar_resources_gcp.R

```diff
@@ -0,0 +1,39 @@
+#' @title Target resources: GCP storage formats
+#' @export
+#' @family resources
+#' @description Create the `gcp` argument of `tar_resources()`
+#'   to specify optional settings to gcp storage formats.
+#'   See the `format` argument of [tar_target()] for details.
+#' @inheritSection tar_resources Resources
+#' @return Object of class `"tar_resources_gcp"`, to be supplied
+#'   to the `gcp` argument of `tar_resources()`.
+#' @param bucket Character of length 1, name of an existing
+#'   gcp bucket to upload and download the return values
+#'   of the affected targets during the pipeline.
+#' @param prefix Character of length 1, "directory path"
+#'   in the S3 bucket where the target return values are stored.
+#' @param verbose Whether to have feedback on the GCS upload process
+#' @examples
+#' # Somewhere in you target script file (usually _targets.R):
+#' tar_target(
```

Comment:
This example is failing because we have not actually created any GCP formats yet. Would you suppress it? Most examples in `targets` only run when a special environment variable is set:

https://github.com/ropensci/targets/blob/0afb8b3a2bf1845aa70398fb7f15f4880033f1a8/R/tar_make_clustermq.R#L31